patches/third_party-blink-public-platform-web_content_settings_client.h.patch

```diff
@@ -14,3 +14,13 @@ index 3f98347f168943b98f9397629d9b046e2bd85a4b..b425394f37c08c1ad79d89b5d5477753
    // Retrieves the client hints that should be attached to the request for the
    // given URL.
    virtual void GetAllowedClientHintsFromSource(const blink::WebURL& url,
+@@ -97,6 +102,9 @@ class WebContentSettingsClient {
+   // interface.
+   virtual bool AllowMutationEvents(bool default_value) { return default_value; }
+ 
++  // Controls whether autoplay is allowed for this frame.
```

Comment:
never add comment lines in patches, but this should use the define as mentioned in the other comments here